### ISSUE TITLE
Innate infravision not shown in character info matrix

### ISSUE THREAD

**Issue by angband-import** [angband/angband#808]
**Reported by ctate@acm.org on 14 Jul 2007 04:12 UTC**
Tested with elf and high-elf characters.

These races have inherent infravision, and this is properly mentioned on the character summary page.  However, this is ''not'' reflected on the character attribute matrix that is the alternate mode of the summary page.  The rightmost column is drawn like this: 

```
      abcdefghijkl\@
Stea.:.............
Sear.:.............
Infra:.............
(etc)
```

It should properly have a '+' character in the Infravision row / '\@' column, like this:

```
      abcdefghijkl\@
Stea.:.............
Sear.:.............
Infra:............+
(etc)
```

Another inherent elf property, Light resistance, is properly annotated in the leftmost column of this same matrix.

**Comment by angband-import** [comment]
**Modified by takkaria on 25 Jul 2007 14:43 UTC**

**Comment by angband-import** [comment]
**Comment by ctate on 28 Jul 2007 23:30 UTC**
Fixed in [c1ee5d1] (SVN r405).

**Comment by angband-import** [comment]
**Modified by takkaria on 17 May 2008 13:26 UTC**
